### PR TITLE
zh: add notes about v1.1

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What is changed, added or deleted? (Required)
Add notes about v1.1

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/notes-tidb-operator-v1.1.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+下面介绍如何将 tidb-cluster chart 管理的组件或者功能切换到 v1.1 完整支持的方式。
+
+### Discovery
+
+Discovery 服务直接由 TiDB Operator 内部生成，不再需要用户做任何配置。
+
+### PD、TiDB、TiKV
+
+在 tidb-cluster chart 中，PD、TiDB、TiKV 配置由 Helm 渲染成 ConfigMap，从 TiDB Operator v1.1 开始，PD、TiDB、TiKV 配置也可以直接在 TiDBCluster CR 中配置，具体配置方法可以参考 [TiDB 集群配置](configure-tidb-cluster.md)。
```
Comment (ran-huang) [inline zh/notes-tidb-operator-v1.1.md]: I already suggest @Yisaer changing the file name in https://github.com/pingcap/docs-tidb-operator/pull/12#issuecomment-601532218
```suggestion
在 tidb-cluster chart 中，PD、TiDB、TiKV 配置由 Helm 渲染成 ConfigMap，从 TiDB Operator v1.1 开始，PD、TiDB、TiKV 配置也可以直接在 TiDBCluster CR 中配置，具体配置方法可以参考 [通过 TidbCluster 配置 TiDB 集群](configure-cluster-using-tidbcluster.md)。
```

File: zh/notes-tidb-operator-v1.1.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+可以参考 [Kubernetes 上的 TiDB 集群监控](monitoring-by-tidb-monitor.md) 创建 TidbMonitor CR，管理 Monitor 组件。
+
+> **注意：**
+>
+> 由于 TiDB Operator 渲染资源的方式和 Helm 渲染资源的方式不同，从 tidb-cluster chart values.yaml 中的配置迁移到 TidbMonitor CR，会引起 Monitor 组件滚动升级。
+
+### Initializer
+
+- 如果在升级到 TiDB Operator v1.1 之前，初始化 job 已经执行，初始化 job 不需要从 tidb-cluster chart 中迁移到 TidbInitializer CR。
+- 如果在升级到 TiDB Operator v1.1 之前，没有执行过初始化 job，也没有修改过 TiDB 服务 root 用户的密码，升级到 TiDB Operator v1.1 之后，需要执行初始化，可以参考 [Kubernetes 上的集群初始化配置](initialize-cluster.md)进行配置。
```
Comment (ran-huang) [inline zh/notes-tidb-operator-v1.1.md]:
```suggestion
- 如果在升级到 TiDB Operator v1.1 之前，没有执行过初始化 job，也没有修改过 TiDB 服务 root 用户的密码，升级到 TiDB Operator v1.1 之后，需要执行初始化，可以参考 [Kubernetes 上的集群初始化配置](initialize-a-cluster.md)进行配置。
```

File: zh/notes-tidb-operator-v1.1.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+> **注意：**
+>
+> 由于 TiDB Operator 渲染资源的方式和 Helm 渲染资源的方式不同，从 tidb-cluster chart values.yaml 中的配置迁移到 TidbCluster CR，会引起 Pump 组件滚动升级。
+
+### Scheduled Backup
+
+升级到 TiDB Operator v1.1 之后，可以通过 BackupSchedule CR 配置定时全量备份：
+
+- 如果 TiDB 集群版本 < v3.1，可以参考 [mydumper 定时全量备份](backup-to-s3.md#定时全量备份)
+- 如果 TiDB 集群版本 >= v3.1，可以参考 [BR 定时全量备份](backup-to-aws-s3-br.md#定时全量备份)
```
Comment (ran-huang) [inline zh/notes-tidb-operator-v1.1.md]:
```suggestion
- 如果 TiDB 集群版本 >= v3.1，可以参考 [BR 定时全量备份](backup-to-aws-s3-using-br.md#定时全量备份)
```

File: zh/notes-tidb-operator-v1.1.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+> **注意：**
+>
+> * BackupSchedule CR mydumper 方式目前只支持备份到 s3、gcs，BR 方式只支持备份到 s3，如果升级之前的定时全量备份是备份到本地 PVC，则升级后不能切换到 CR 方式管理。
+> * 如果切换到 CR 方式管理，请删除原有定时全量备份的 Cronjob，以防止重复备份。
+
+### Drainer
+
+- 如果在升级到 TiDB Operator v1.1 之前，没有部署 Drainer，如果要新部署，可以参考 [Drainer 部署](maintain-tidb-binlog.md#部署多个-drainer)。
```
Comment (ran-huang) [inline zh/notes-tidb-operator-v1.1.md]:
```suggestion
- 如果在升级到 TiDB Operator v1.1 之前，没有部署 Drainer，现在需要新部署，可以参考 [Drainer 部署](maintain-tidb-binlog.md#部署多个-drainer)。
```

File: zh/notes-tidb-operator-v1.1.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+- 如果在升级到 TiDB Operator v1.1 之前，已经部署 Drainer，建议直接用 kubectl 管理。
+
+### TiKV Importer
+
+- 如果在升级到 TiDB Operator v1.1 之前，没有部署 TiKV Importer，如果要新部署，可以参考 [TiKV Importer 部署](restore-data-using-tidb-lightning.md#部署-tikv-importer)。
```
Comment (ran-huang) [inline zh/notes-tidb-operator-v1.1.md]:
```suggestion
- 如果在升级到 TiDB Operator v1.1 之前，没有部署 TiKV Importer，现在需要新部署，可以参考 [TiKV Importer 部署](restore-data-using-tidb-lightning.md#部署-tikv-importer)。
```

**Comment by ran-huang** [comment]
@DanielZhangQD Please address comments and resolve conflicts.
@shuijing198799 PTAL~

File: zh/notes-tidb-operator-v1.1.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+| TiDB Cluster (PD, TiDB, TiKV) | [TidbCluster CR](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.html) |
+| TiDB Monitor | [TidbMonitor CR](https://github.com/pingcap/tidb-operator/blob/master/manifests/monitor/tidb-monitor.yaml) |
+| TiDB Initializer | [TidbInitializer CR](https://github.com/pingcap/tidb-operator/blob/master/manifests/initializer/tidb-initializer.yaml) |
+| Scheduled Backup | [BackupSchedule CR](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-schedule-aws-s3-br.yaml) |
+| Pump | [TidbCluster CR](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.html) |
+| Drainer | [tidb-drainer chart](https://github.com/pingcap/tidb-operator/tree/master/charts/tidb-drainer) |
+| Importer | [tikv-importer chart](https://github.com/pingcap/tidb-operator/tree/master/charts/tikv-importer) |
+
+- tidb-cluster chart 仍然会继续发布，但是不再添加新功能，比如组件 TLS（目前只是部分支持）、TiKV 和 TiDB 自动伸缩等功能。
+- 通过 v1.0.x TiDB Operator 部署的 TiDB 集群，在 TiDB Operator 升级到 v1.1 之后，仍然可以通过 v1.1 版本的 tidb-cluster chart 升级和管理 TiDB 集群。
```
[the remark this answers is not in the document]
Comment (DanielZhangQD) [reply zh/notes-tidb-operator-v1.1.md]: The description here is for the tidb clusters deployed with v1.0.x TiDB Operator.

**Comment by DanielZhangQD** [comment]
@ran-huang comments addressed, PTAL again.
@weekface @cofyc @Yisaer Please help review this PR.

**Pull request merged**